### PR TITLE
Unify our timeout.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/138/
Test PASSed.

File: theblues/charmstore.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 class CharmStore(object):
     """A connection to the charmstore."""
 
-    def __init__(self, url, macaroons=None, timeout=None, verify=True):
+    def __init__(self, url, macaroons=None, timeout=3.05, verify=True):
```
Comment (bac) [inline theblues/charmstore.py]: Would be nice if the magic number of 3.0.5 were defined in one place rather than having a constant everywhere.

**Comment by bac** [comment]
:+1:

File: theblues/charmstore.py (hunk 1 above)
Comment (bac) [inline theblues/charmstore.py]: maybe utils.DEFAULT_TIMEOUT

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/139/
Test PASSed.

File: theblues/jem.py [hunk 2]
```diff
@@ -27,7 +31,13 @@ def fetch_macaroon(self):
             # We don't use make_request b/c we don't want the request to be
             # fully handled. This lets us get the macaroon out of the request
             # and keep it.
-            response = requests.get("{}model".format(self.url))
+            url = "{}model".format(self.url)
+            response = requests.get(url)
```
Comment (jrwren) [inline theblues/jem.py]: Doesn't the timeout need to be specified here?  request.get(url, timeout=self.timeout) ?

**Comment by jrwren** [comment]
👍   One change iff it is required.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/140/
Test PASSed.

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/theblues-lib-merge

**Pull request merged**